src/Assignment6/Bankingsystem/Program.cs

```diff
@@ -0,0 +1,121 @@
+                        do
+                        {
+                            accountnumber = Inputvalidators.StringValidator();
+                        }
+                        while (Inputvalidators.AccountNumberValidate(accountnumber) == false);
+                        BankAccount savingsaccount = new SavingsAccount(accountnumber);
+                        do
+                        {
+                            Console.WriteLine("\nChoose The account option:\n1.Deposit\n2.Withdrawal\n3.Check Balance\n4.Exit");
+                            int accountoptions = Inputvalidators.IntegerValidator();
+                            switch (accountoptions)
+                            {
+                                case (int)EnumOptions.AccountOptions.Deposit:
+                                    Console.WriteLine("Enter the amount to be deposited:");
+                                    float depositamount = Inputvalidators.FloatValidator();
+                                    savingsaccount.DepositAmount(depositamount);
+                                    break;
+
+                                case (int)EnumOptions.AccountOptions.Withdrawal:
+                                    Console.WriteLine("Enter the amount to be Withdrawn:");
+                                    float withdrawamount = Inputvalidators.FloatValidator();
+                                    savingsaccount.WithdrawAmount(withdrawamount);
+                                    break;
+
+                                case (int)EnumOptions.AccountOptions.CheckBalance:
+                                    savingsaccount.CheckBalance();
+                                    break;
+
+                                case (int)EnumOptions.AccountOptions.Exit:
+                                    x = false;
+                                    break;
+
+                                default:
+                                    Console.WriteLine("Choose appropriate option.");
+                                    break;
+                            }
+                        }
+                        while (x == true);
+                        break;
+
+                    case (int)EnumOptions.AccountType.Checkingaccount:
+                        Console.WriteLine("Enter the account number:");
+                        do
+                        {
+                            accountnumber = Inputvalidators.StringValidator();
+                        }
+                        while (Inputvalidators.AccountNumberValidate(accountnumber) == false);
+                        BankAccount checkingaccount = new CheckingAccount(accountnumber);
+                        do
+                        {
+                            Console.WriteLine("\nChoose the account option:\n1.Deposit\n2.Withdrawal\n3.CheckBalance\n4.Exit");
+                            int accountoptions1 = Inputvalidators.IntegerValidator();
```

Comment:
Don't use number suffixes, try to use appropriate names.

Reply:
Number suffixes are removed.